### DISCUSSION TITLE
TrinityCore wotlk_classic = 3.4.3.52237

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#727] [Q&A]
trickerer, привет!
Можно приметить патч с ботами к ветке wotlk_classic = 3.4.3.52237?

**Comment by trickerer** [comment]
Не знаю, чем она отличается-то вообще? Зачем она нужна и что в ней такого особенного?

**Comment by xBarselonx** [comment]
В чем особенность тоже не знаю. Я так понял они сделали это под новую версию клиента wotlk 3.4.3.

**Comment by trickerer** [comment]
Что под новый клиент это понятно. Я им не пользуюсь и говорить об этой ветке пока рано, сразу могу сказать что она скорее всего идёт лесом.

**Comment by xBarselonx** [comment]
Да мне в принципе тоже не интересно. Меня все устраивает в версии 3.3.5а.

**Comment by Effectgod** [comment]
кто-то может поделится client wotlk_classic ru-ru?
